packages/opentelemetry-plugin-postgres/package.json

```diff
@@ -42,11 +43,14 @@
   "devDependencies": {
     "@types/mocha": "^5.2.7",
     "@types/node": "^12.6.9",
+    "@types/pg": "^7.11.2",
+    "@types/shimmer": "^1.0.1",
     "codecov": "^3.5.0",
     "gts": "^1.1.0",
```

Comment:
Please use `gts` version `^1.0.0` in order to be consistent with other packages dependencies.